test/e2e/profiles/kratos.base.yml

```diff
@@ -25,9 +25,11 @@ selfservice:
 
 serve:
   public:
-    # We are using a proxy to make cypress not fail with different origins which is why we use this URL here.
+    # We are using a proxy to make cypress not fail with different origins.
+    # the proxy sets the X-FORWARDED-HOST header which allows the base_url here to be set to
+    # the kratos instance
     # See ../proxy
-    base_url: http://localhost:4455/
+    base_url: http://localhost:4433/
```

Comment:
nvm, just had a lot of cases where the URL wouldn't be crafted correctly e.g. /schemas would be under the kratos public url (localhost:4433) instead of the proxy url (localhost:4455). Reverted this change in the config but still kept the `x-forwarded-host` headers from the proxy side.